tests/Kernel/ValueObject/ImageTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\ValueObject;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\oe_theme\ValueObject\ImageValueObject;
+use Drupal\Tests\token\Kernel\KernelTestBase;
+
+/**
+ * Test image value object with image field type.
+ */
+class ImageTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'image',
+    'file',
+    'entity_test',
+    'field',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installEntitySchema('file');
+    $this->installEntitySchema('entity_test');
+
+    $this->installSchema('file', ['file_usage']);
+
+    $this->installConfig(['field', 'system']);
+
+    // Create a field with settings to validate.
+    $field_storage = FieldStorageConfig::create([
+      'entity_type' => 'entity_test',
+      'field_name' => 'field_image',
+      'type' => 'image',
+    ]);
+    $field_storage->save();
+    FieldConfig::create([
+      'field_storage' => $field_storage,
+      'bundle' => 'entity_test',
+    ])->save();
+
+    // Copy file in public files to use it for styling.
+    \Drupal::service('file_system')->copy(__DIR__ . '/../../fixtures/example_1.jpeg', 'public://example_1.jpg');
+  }
+
+  /**
+   * Test the image value object has the correct style applied.
+   */
+  public function testFromStyledImageItem() {
```

Comment:
Since we are adding tests for the ImageValueObject, lets add a test for fromImageItem as well.

Reply:
It is tested in here: https://github.com/openeuropa/oe_theme/blob/2.x/tests/Unit/ValueObject/ImageValueObjectTest.php#L107